File: URI Programs/C/1047.c
#include <stdio.h>

int main(){
	int initialH, finalH, initialM, finalM, durationH, durationM;
	scanf("%d %d %d %d", &initialH, &initialM, &finalH, &finalM);
	if (finalH >= initialH){
		durationH = finalH - initialH;
	} else {
		durationH = 24 - initialH + finalH;
	}
	if (finalM >= initialM){
		durationM = finalM - initialM;
	} else {
		durationH--;
		durationM = 60 - initialM + finalM;
	}
	if ((initialH == finalH) && (initialM == finalM)){
		durationH = 24;
		durationM = 0;
	}
	printf("O JOGO DUROU %d HORA(S) E %d MINUTO(S)\n", durationH, durationM);
	return 0;
}

File: URI Programs/Python/1047.py
h1,m1,h2,m2 = map (int, input().split())

if h1 < h2:
  h = h2 - h1
else:
  h = 24 - h1 + h2
if m1 <= m2:
  m = m2 - m1
else:
  h = h -1
  m = 60 - m1 + m2

print("O JOGO DUROU", h, "HORA(S) E", m, "MINUTO(S)")